src/main/groovy/com/ceilfors/transform/gq/SingletonCodeFlowManager.groovy

```diff
@@ -23,35 +23,50 @@ enum SingletonCodeFlowManager implements CodeFlowListener {
 
     INSTANCE;
 
-    CodeFlowListener codeFlowListener = new GqFile(createGqFile(), true)
+    private CodeFlowPrinter codeFlowPrinter
 
     private File createGqFile() {
         // By default using "/tmp" instead of using java.io.tmpdir for better user usability
         return new File(System.getProperty("GQTMP", "/tmp"), "gq")
```

Comment:
Will convert to constant. Also just remembered that Java property should have looked like `gq.tmp`.